### ISSUE TITLE
高级样式不支持Android吗？

### ISSUE THREAD

**Issue by pdwinterdev** [geek-prince/react-native-page-scrollview#4]
在Android设备上运行
<PageScrollView
                    style={{ width : SCREEN_WIDTH, height : SCREEN_WIDTH / 16 * 9 }}
                    builtinStyle='sizeChangeMode'
                    builtinWH={{ width : 300, height : 300 / 16 * 9 }}
                    imageArr={this.imgArr}/>
两边小部分空白的

**Comment by geek-prince** [comment]
应该是可以的,当时写的时候在两个系统上都测试过,虽然样式有略微的区别,不过大体上是一样的

**Comment by lglong519** [comment]
高级样式android好卡

**Comment by geek-prince** [comment]
是的,现在这个是会有些卡,因为写完之后一直有其他事情就没有对性能进行优化.不过最新公司项目会用到这个插件,我也准备过年之后回来优化性能之后更新一版
